docker/Nginx/default.conf

```diff
@@ -0,0 +1,21 @@
+server {
+    listen 80;
```

Comment:
Also introduce https configuration as of now your Ngnix is configured to work on only http mode.